### ISSUE TITLE
WCAG 2.1: btn-primary focus fail

### ISSUE THREAD

**Issue by shawnthompson** [wet-boew/GCWeb#1371]
The primary button hover / focus colour change does not meet WCAG 2.1 [Success Criterion 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Comment by zzucco** [comment]
Adding some more information here:
The button default colour is #335075. 
When focused, it becomes #1c578a
The contrast ratio is 1.09. It must be at least 3.

The actual process of determining the focused colour is somewhat all over the place. It seems we are taking the colour #2572b4 ([L26 here](https://github.com/wet-boew/wet-boew/blob/master/src/_variables.scss)) and bootstrap darkens it by 23% - not 100% sure this is how it's working, though. 

Replacing that colour with #a9ceed should satisfy WCAG, but then the white text on the button gets hard to read.

We would need to change the text colour as well as the background colour, or consider a different solution such as an outline on hover/focus.

**Comment by shawnthompson** [comment]
Sorry I just noticed that this actually fails in the wet-boew theme too so maybe this should be moved to wet-boew.

**Comment by duboisp** [comment]
@shawnthompson feel free to open an issue in the wet-boew repo too.
